app/util/legUtils.js

```diff
@@ -3,6 +3,14 @@ import get from 'lodash/get';
 import { BIKEAVL_UNKNOWN } from './vehicleRentalUtils';
 import { getRouteMode } from './modeUtils';
 
+/**
+ * Get time as  milliseconds since the Unix Epoch
+ */
+export function legTime(lt) {
+  const t = lt.estimated?.time || lt.scheduledTime;
+  return Date.parse(t);
+}
```

Comment:
Quite often we use this for moment. I wonder if it makes any difference to create a method that directly parses the ISO 8601 time to moment.

Reply:
I added new util which returns an actual time string in the  form UI frequently uses.